### DISCUSSION TITLE
Can envd specfify `pip` path? I am using `/databricks/python3/bin/pip` as my pip python path to run python job.

### DISCUSSION THREAD

**Discussion by Future-Outlier** [tensorchord/envd#1798] [Q&A] [answered]
Please help, thanks a lot.

**Comment by kemingy** [comment, marked as answer]
Do you mean the `pip` in the envd environment? It's not recommended to reuse the host Python environment. But you can certainly mount it into the envd environment.

```python
runtime.mount(host_path="xxx", envd_path="xxx")
```

BTW, you will need to mount the whole Python environment, not just the `pip` binary file.

**Reply by Future-Outlier** [reply]
Thanks a lot, I will ask more questions if I face problems.
